starcluster/commands/start.py

```diff
@@ -264,8 +264,8 @@ def execute(self, args):
                     "Cannot use --dns-prefix when the cluster tag contains "
                     "a dot.")
             scluster.dns_prefix = tag
-        if self.opts.dns_sufix:
-            scluster.dns_sufix = tag
+        if self.opts.dns_suffix:
+            scluster.dns_suffix = self.opts.dns_suffix
```

Comment:
This PR changes the behaviour of the --dns-suffix option. The description says

    Suffix dns names of all nodes in the cluster with the cluster tag.

Here is doesn't use the tag, it uses the --dns-suffix value instead.